internal/core/testing.go

```diff
@@ -386,6 +388,7 @@ func Test(config *TestConfig) func(t *testing.T) {
 				Logger:           log,
 			})
 
+			log.Debugf("Store in meta (with key CmdResult): %s\n", result)
```

Comment:
```suggestion
			log.Debugf("Command %s give result: %s\n", args, result)
```

Reply:
The result will almost never be readable because it's a complex GoStruct. Not sure If we shoulod print it. Maybe json_encode it ?